website/docs/reference/deploy/environment-import-export.mdx

```diff
@@ -0,0 +1,110 @@
+
+<Figure caption="Search-driven export screen with WYSIWYG mode."  img="/img/export.png"/>
+
+## Import
+
+Import is a 3 stage process designed to be efficient and error-resistant.
+
+### Import stages
+
+* **upload** - you can upload previously exported JSON file or copy-paste export data from the exported JSON file into the code editor
+* **validation** - you will get feedback on any errors or warnings before you do the actual import. This makes sure your feature flags configurations. You will not be able to finish the import if you have errors. Warnings don't stop you from importing.
+* **import** - the actual import that creates a new configuration in the target environment or creates a [change request](../change-requests.md) when the environment has change requests enabled
+
+![Three-staged import process](/img/import.png)
+
+### Import rejection
+
+An import will be rejected when there's a possibility of data corruption.
+
+#### Context fields
+
+The exported and imported context fields have the same name but the target environment doesn't support some of the source environment context field values. If the context fields don't exist in the target environment they will be created.
+
+#### Segments
+
+The target Unleash instance doesn't have a matching segment. The actual constraints in the segment can be different. The matching is done by the segment name.
+
+#### Custom strategies
+
+The exported strategies include custom strategies that don't exist in the target Unleash instance. The matching is done by the strategy name.
+
+#### Existing features
+
+You're trying to import a feature that already exists in a different project on the target instance.
+
+#### Pending change request
+
+The target project and environment have a pending change request (applies to enterprise customers with change requests enabled for a given project and environment).
+
+### Import warnings
+
+Import validation will warn you when:
+* the features you are trying to import are already archived in the target instance in any project. Those features won't be imported again.
+* the strategies you are trying to import are custom and may have different [configuration parameters](../custom-activation-strategies.md#parameters). Those strategies will be imported anyway.
+
+### Required permissions
+
+To import features, you will **always** require the **update feature toggles** permission.
+Additionally, depending on how the actions your import job would trigger, you may also require any of the following permissions:
+* **Create feature toggles**: When the import would create new features
+* **Update tag types**: when the import would create new tag types
+* **Create context fields**: when the import would create new context fields
+* **Create activation strategies**: when the import would add (or is this "create custom activation strategies"? custom or standard?) activation strategies to a feature and change requests are disabled
```

Comment:
I think you missed the question I added for you on this line 😅  Could you clarify whether this is for creating custom strategies or adding a strategy to a feature?

Reply:
Okay. That's ... pretty confusing to me. But okay, I'll make a suggestion for it.